server/src/main/java/com/epam/aidial/core/server/controller/FileMetadataController.java

```diff
@@ -55,7 +55,7 @@ protected Future<?> handle(ResourceDescriptor resource, boolean hasWriteAccess)
             } catch (Exception ex) {
                 log.error("Failed to list files", ex);
                 context.respond(HttpStatus.INTERNAL_SERVER_ERROR,
-                        "Failed to list files by path %s/%s".formatted(resource.getBucketName(), resource.getOriginalPath()));
+                        "Failed to list files by path %s".formatted(resource.getUrl()));
```

Comment:
Why to change?

Reply:
I removed originalPath since it's used just here and very similar to what has been changed

Reply:
Sorry, misread what was changed. This is good change.